docs/section-6-contributing.md

```diff
@@ -48,48 +48,42 @@ This will create the `tree-sitter` CLI executable in the `target/release` folder
 Before you can run the tests, you need to clone some grammars that are used for testing:
 
 ```sh
-script/fetch-fixtures
+cargo xtask fetch-fixtures
 ```
 
 To test any changes you've made to the CLI, you can regenerate these parsers using your current CLI code:
 
 ```sh
-script/generate-fixtures
+cargo xtask generate-fixtures
 ```
 
 Then you can run the tests:
 
 ```sh
-script/test
+cargo xtask test
 ```
 
 Similarly, to test the WASM binding, you need to compile these parsers to WASM:
 
 ```sh
-script/generate-fixtures-wasm
-script/test-wasm
+cargo xtask generate-fixtures --wasm
+cargo xtask test-wasm
 ```
 
 ### Debugging
 
-The test script has a number of useful flags. You can list them all by running `script/test -h`. Here are some of the main flags:
+The test script has a number of useful flags. You can list them all by running `cargo xtask test -h`. Here are some of the main flags:
 
 If you want to run a specific unit test, pass its name (or part of its name) as an argument:
 
 ```sh
-script/test test_does_something
+cargo xtask test test_does_something
 ```
 
 You can run the tests under the debugger (either `lldb` or `gdb`) using the `-g` flag:
 
 ```sh
-script/test test_does_something -g
-```
-
-Part of the Tree-sitter test suite involves parsing the _corpus_ tests for several different languages and performing randomized edits to each example in the corpus. If you just want to run the tests for a particular _language_, you can pass the `-l` flag. And if you want to run a particular _example_ from the corpus, you can pass the `-e` flag:
```

Comment:
Hm the `-e` flag still exists, so we could retain this info minus the sentence mentioning the `-l` flag.

Reply:
Hi, I've played with these two options a bit and discovered that maybe both `-l` and `-e` should remain.

For `-l`, the language filter was not in effect for `cli::corpus_test::test_language_corpus`. So I added an early return if the filter is set and the language is not the filtered language. The `xtask test` would set the `TREE_SITTER_LANGUAGE` environment variable to enable language filtering.

For `-e`, the correct environment variable should `TREE_SITTER_EXAMPLE_INCLUDE` rather than `TREE_SITTER_EXAMPLE`.

I've fixed both problems and tested that the environment variables are in effect for the corpus tests.

Reply:
Yeah that's great. `-e` was already there but `-l` was removed before the migration to `xtasks`. Thank you for the fixes!